### DISCUSSION TITLE
Control of LED´s color and state

### DISCUSSION THREAD

**Discussion by MegZamani** [koenhendriks/ha-button-plus#47] [Ideas]
Hi,

Will there be any option to control the RGB LED´s for each button in Home Assistant?

/Meg

**Comment by koenhendriks** [comment]
Hi!

I think that should be taken up in #33, so yes it's on the roadmap 😁

**Comment by roodbaard80** [comment]
Thanks for this great integration already. It really made it a lot easier to get started.
But adding the LED's would really be a nice addition.
